src/view/ThemeSettings.js

```diff
@@ -100,7 +100,24 @@ define(function (require, exports, module) {
                 var attr = $target.attr("data-target");
                 newSettings[attr] = $target.is(":checked");
             })
-            .on("input", "[data-target]:text", function () {
+            .on("input", "#font-size-input", function () {
+                var $target = $(this);
+                var $input = $target.val();
+                var validInput = /^[\d\.]+(p|px|e|em){0,1}$/g;
+                var btn = document.getElementById("done");
```

Comment:
add `document` to the globals please: https://github.com/adobe/brackets/blob/master/.eslintrc.json#L52